src/device.rs

```diff
@@ -22,68 +21,158 @@ impl types::StaticType for Device {
 }
 
 impl Device {
+    /// Determines the name of the device.
     pub fn get_name(&self) -> Option<String> {
         unsafe {
             from_glib_none(ffi::gdk_device_get_name(self.to_glib_none().0))
         }
     }
 
+    /// Returns the vendor ID of this device, or NULL if this information couldn't be obtained. This
+    /// ID is retrieved from the device, and is thus constant for it.
+    ///
+    /// This function, together with gdk_device_get_product_id(), can be used to eg. compose GSettings
+    /// paths to store settings for this device.
+    ///
+    /// ```ignore
+    /// fn get_device_settings(device: &Device) -> GSettings {
+    ///     GSettings *settings;
+    ///     GdkDevice *device;
+    ///     gchar *path;
+    /// 
+    ///     let vendor = device.get_vendor_id().unwrap();
+    ///     let product = device.get_product_id().unwrap();
+    /// 
+    ///     let path = format!("/org/example/app/devices/{}:{}/", vendor, product);
+    ///     g_settings_new_with_path(DEVICE_SCHEMA, path);
+    /// }
+    /// ```
+    #[cfg(feature = "gdk_3_16")]
```

Comment:
The current syntax is `cfg(gdk_3_16)` and that regex is how I'd do the conversion.